### PR TITLE
update graph CI

### PR DESCRIPTION
**Pull request by ilaner**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation
- [ ] [VS Code Extension](https://docs.gitlab.com/ee/ci/yaml/#tags)
  - [ ] Functionality implemented in the extension - <u>\<link-to-pr-here\></u>
  - [ ] N/A - Functionality cannot be implemented in the extension because <u>\<add-reason-here\></u>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -195,7 +195,7 @@ jobs:
               mkdir graph-test-results
               neo4j start
               # poll for neo4j status until available
-              while ! neo4j status; do sleep 1; done
+              while ! curl --fail http://127.0.0.1:7474 &> /dev/null; do sleep 1; done
```
Comment (dorschw) [inline .circleci/config.yml:198]: Please add a log message on each iteration, so we know what's going on

File: .circleci/config.yml [hunk 2]
```diff
@@ -356,6 +356,35 @@ jobs:
...
+              name: Test Update graph
+              when: always
+              command: |
+                source $(poetry env info --path)/bin/activate
+                cd content
+                demisto-sdk -v
+                mkdir ./tmp
+                neo4j start
+                # poll for neo4j status until available
+                while ! curl --fail http://127.0.0.1:7474 &> /dev/null; do sleep 1; done
```
Comment (dorschw) [inline .circleci/config.yml:379]: here too

File: .circleci/config.yml (hunk 1 above)
Comment (ilaner) [reply .circleci/config.yml:198]: I can remove the `&> /dev/null` to print the original output, which is a connection error. Personally, I think it just spams the log and will make people think that there's an error even though everything works as expected. The connection should take at most 10 seconds.

File: .circleci/config.yml [hunk 3]
```diff
@@ -356,6 +356,36 @@ jobs:
               demisto-sdk create-content-artifacts -a ./tmp
         - store_artifacts:
             path: content/tmp
+  update-graph:
```
Comment (dantavori) [inline .circleci/config.yml:359]:
```suggestion
  test-update-graph:
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -356,6 +356,36 @@ jobs:
...
+    resource_class: large
+    steps:
+      - checkout
+      - <<: *install_poetry
+      - <<: *install_neo4j
+      - attach_workspace:
+          at: ~/project
+      - <<: *install_demisto_sdk
+      - run:
+          name: Test Update graph
```
Comment (dantavori) [inline .circleci/config.yml:371]:
```suggestion
          name: Run Update Graph Twice
```

**Review by dantavori — APPROVED** [review]

**Pull request merged**